controllers/userDeviceInfo.js

```diff
@@ -0,0 +1,25 @@
+const UserDeviceInfoModel = require("../models/userDeviceInfo");
+const { SOMETHING_WENT_WRONG } = require("../constants/errorMessages");
+
+const storeUserDeviceInfo = async (req, res) => {
+  try {
+    const userJson = {
+      userId: req.body.userId,
+      deviceType: req.body.deviceType,
```

Comment:
without any means for validation won't we be able to ass anyone's device info to anyone else?
I can pass anyone's user id in request body

example I run Postman and pass Prakash's user id and my own device data.

Reply:
yeah I agree, we are designing this for the MVP, and later will enhance it.